CBLClient/Apps/CBLTestServer-Java-Desktop/build_java_desktop_testserver_app.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+EDITION=$1
+VERSION=$2
+BUILD_NUM=$3
+
+FAIL=0
+if [ -z "${ARTIFACTS_DIR}" ] || [ ! -d "${ARTIFACTS_DIR}" ]; then
```

Comment:
ARTIFACTS_DIR is an environment variable, jenkins build job defines it. if run this script locally, you may set this env pointing to anywhere on your local drive.

Reply:
FWIW, using environmental variables makes it a little bit harder to understand and debug these shell scripts.  It seems like good practice to use names, inside the script, that are not likely to have been defined in the environment, and to pass any value that you need, as a script parameter

Reply:
@bmeike I agree with you, but this code is copied from whatever android build script. I am going to create a ticket to convert environment variables to regular params for all 3 platforms (android/java/java-ws). will keep it as-it in this PR (because jenkins jobs also need to be changed and tested) and fix them all together.